main/client/features/chart-birth-month/birth-month-chart-ctrl.js

```diff
@@ -0,0 +1,169 @@
+angular.module('app').controller('birthMonthCtrl', function($scope, $rootScope, birthMonthService) {
+
+    //Load totalBirthsPerMonth From Service
+    $scope.totalBirthsPerMonth = birthMonthService.totalBirthsPerMonth;
+
+    //Load TotalBirthsPer Generation from Service
+    $scope.totalBirthsPerGeneration = birthMonthService.totalBirthsPerGeneration;
+
+    //MONTH VIEW LOGIC ====================================================
+	//MONTH VIEW LOGIC ====================================================
+	//MONTH VIEW LOGIC ====================================================
+
+    $scope.janChecked = function() {
```

Comment:
Maybe see if you can condense these into one function if you have time. Lots of repeated logic.